src/dscanner/analysis/properly_documented_public_functions.d

```diff
@@ -51,6 +51,49 @@ final class ProperlyDocumentedPublicFunctions : BaseAnalyzer
 		postCheckSeenDdocParams();
 	}
 
+
+
+    override void visit(const UnaryExpression decl)
+    {
+        const IdentifierOrTemplateInstance iot = safeAccess(decl)
+            .functionCallExpression.unaryExpression.primaryExpression
+            .identifierOrTemplateInstance;
+
+        // enforce(condition);
+        if (iot && iot.identifier.text == "enforce")
+        {
+            Type tt = new Type;
+            tt.type2 = new Type2;
+            tt.type2.typeIdentifierPart = new TypeIdentifierPart;
+            tt.type2.typeIdentifierPart.identifierOrTemplateInstance =
+                new IdentifierOrTemplateInstance;
+            tt.type2.typeIdentifierPart.identifierOrTemplateInstance.identifier =
+                Token(tok!"identifier", "Exception", 0, 0, 0);
+            thrown ~= tt;
+        }
+        // enforce!Type(condition);
+        else if (const TemplateSingleArgument tsa = safeAccess(iot).templateInstance
```

Comment:
This branch doesn't seem to check for "enforce".

Reply:
Yes fixed and thanks for finding this.